[README] Document support for installation via Accio

Dschee opened the pull request:
This documents support for the new SwiftPM based dependency manager [Accio](https://github.com/JamitLabs/Accio).

Please note that this project is part of Accio's official [integration tests](https://github.com/JamitLabs/Accio/tree/work/1000-frameworks/Demo/Shared/AppDependencies) within the [Demo project](https://github.com/JamitLabs/Accio/tree/work/1000-frameworks/Demo).

LucianoPAlmeida commented:
Hey @Dschee :) 
Can you update this to look like the one on Moya? I think the explanation there is more clear.

Dschee commented:
@LucianoPAlmeida Sure, no problem, I just rebased my branch and updated the SwiftPM and Accio sections accordingly. Note that I completed the target integration instructions in the SwiftPM section and pointed to accio for building for iOS & co.

I hope it looks good now!

SD10 commented:
Thank you for contributing to SwifterSwift! I've invited you to join the SwifterSwift GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md). Feel free to reach out if you have any questions! 😃
